src/components/30-organisms/testimonials/story.js

```diff
@@ -36,7 +36,7 @@ story.add('Testimonials', () => {
       title="${title}"
       subtitle="${subtitle}"
     >
-     <div>
+      <div class="showallinline_ie_bugfix">
```

Comment:
dtto, name does not follow our css rules: `o-testimonials__showallinline__ie__bugfix`